PendingReleaseNotes

```diff
@@ -139,6 +139,24 @@
 * CEPHFS: After recovering a Ceph File System post following the disaster recovery
   procedure, the recovered files under `lost+found` directory can now be deleted.
 * core: cache-tiering is now deprecated.
+* mClock Scheduler: The mClock scheduler (introduced in Quincy) has undergone
+  significant changes both user visible and under the hood to address the slow
+  backfill issue. Some important changes are:
+  * The 'balanced' profile is set as the default mClock profile.
+  * QoS parameters like reservation and limit are now specified in terms of a
+    fraction (range: 0.0 to 1.0) of the OSD's IOPS capacity.
+  * The cost parameters (osd_mclock_cost_per_io_usec_* and
+    osd_mclock_cost_per_byte_usec_*) are removed. The cost of an operation is now
+    determined using the random IOPS and maximum sequential bandwidth capability
+    of the OSD's underlying device.
+  * Degraded object recovery is given higher priority when compared to misplaced
+    object recovery. Therefore, backfilling operations with the 'balanced' and
+    'high_client_ops' mClock profiles may progress slower than what was seen with
+    the 'WeightedPriorityQueue' (WPQ) scheduler.
```

Comment:
We should explain the reason behind this change so as to not scare users.

Reply:
...misplaced object recovery because degraded objects present a data safety issue not present with objects that are merely misplaced.

Reply:
Changed.

Reply:
Note to Zac of the future: "Degraded object recovery is now given higher priority than misplaced object recovery. This means that backfilling operations that have the ``balanced`` and ``high_client_ops`` mClock profiles might be slower than backfilling operations were when the ``WeightedPriorityQueue`` (WPQ) scheduler was used."